examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p>The table below shows mean and std of evaluated accuracy for each experiment averaged over its repetitions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>